scripts/vars.sh

```diff
@@ -8,25 +8,34 @@ PEER_PORT=26656
 
 # DENOMS
 IBC_STRD_DENOM='ibc/FF6C2E86490C1C4FBBD24F55032831D2415B9D7882F85C3CC9C2401D79362BEA'
+IBC_STRD_DENOM_JUNO='ibc/FF6C2E86490C1C4FBBD24F55032831D2415B9D7882F85C3CC9C2401D79362BEA'
+IBC_STRD_DENOM_OSMO='ibc/FF6C2E86490C1C4FBBD24F55032831D2415B9D7882F85C3CC9C2401D79362BEA'
 IBC_ATOM_DENOM='ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
+IBC_JUNO_DENOM='ibc/EFF323CC632EC4F747C61BCE238A758EFDB7699C3226565F7C20DA06509D59A5' 
+IBC_OSMO_DENOM='ibc/13B2C536BB057AC79D5616B8EA1B9540EC1F2170718CAFF6F0083C966FFFED0B'  
 ATOM_DENOM='uatom'
+JUNO_DENOM='ujuno'
+OSMO_DENOM='uosmo'
 STRD_DENOM='ustrd'
 STATOM_DENOM="stuatom"
+STJUNO_DENOM="stujuno"
+STOSMO_DENOM="stuosmo"
 
-BLOCK_TIME='5s'
+# CHAIN PARAMS
+BLOCK_TIME='1s'
 DAY_EPOCH_INDEX=1
-DAY_EPOCH_DURATION="100s"
+DAY_EPOCH_DURATION="40s"
 STRIDE_EPOCH_INDEX=2
-STRIDE_EPOCH_DURATION="40s"
-UNBONDING_TIME="200s"
+STRIDE_EPOCH_DURATION="20s"
```

Comment:
Just sanity checking, is this enough time? Never have run things reliably with `stride_epoch` below 40s but totally possible it works! Thinking the main constraint is the time it takes for daisy chained ICQ => ICA for reinvestment must fit into 80% of this time (per the ICQ buffer). Should in theory work (ICA/ICQ each only ~2-3s)

Reply:
I grabbed those params from the current localnet settings (and it looks like those are the current settings on main)! I have no prior on what these should be though, so I can lengthen them if you think it makes sense